src/vaults/mainnet.json

```diff
@@ -2581,6 +2588,42 @@
       "description": "Acquired by depositing liquidity into the OpenState - osBGT | sWBERA Pool on Kodiak",
       "owner": "OpenState",
       "action": "Stake osBGT / sWBERA"
+    },
+    {
+      "stakingTokenAddress": "0xc2605ed80880bd6b1523d52aef8d624ed468a935",
+      "vaultAddress": "0x3d1c173460fa24e53d68b31c39d85d874348ed8b",
+      "name": "LP-iBGT on Pendle",
+      "protocol": "Pendle",
+      "categories": ["defi"],
+      "logoURI": "https://raw.githubusercontent.com/berachain/metadata/main/src/assets/vaults/0x3d1c173460fa24e53d68b31c39d85d874348ed8b.png",
```

Comment:
Image not found, needs to be in cloudinary

Reply:
its because they aren't in the main branch yet, personally i find this easier than cloudinary but i will add them and update